host-ext/nacl_src/kiwi.cc

```diff
@@ -246,6 +246,12 @@ class KiwiInstance : public pp::Instance {
         return false;
     }
 
+    /* Recieves and processes initialization information */
+    void SocketParseInitInformation(const char* data, int datalen) {
```

Comment:
I think that might present problems (adding a field to the `initinfo` struct doesn't necessarily increase `datalen`). 
I was thinking of using the server_version_ variable for versioning instead.